src/sources/dnstap/parser.rs

```diff
@@ -600,26 +594,26 @@ impl<'a> DnstapParser<'a> {
 
         self.insert(
             self.event_schema.dns_query_header_schema().question_count(),
-            header.question_count as i64,
+            header.question_count,
         );
 
         self.insert(
             self.event_schema.dns_query_header_schema().answer_count(),
-            header.answer_count as i64,
+            header.answer_count,
         );
 
         self.insert(
             self.event_schema
                 .dns_query_header_schema()
                 .authority_count(),
-            header.authority_count as i64,
```

Comment:
I removed most of these `as i64`s because `Event` already implements `From` for most numeric types (which just calls `as i64`).